pkg/images/entities.go.new

```diff
@@ -0,0 +1,5 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
```

Comment:
seems this .new file was generated by header-check and entities.go not removed from git